advanced_sheet_label/layouts.py

```diff
@@ -108,6 +108,17 @@ def __str__(self) -> str:
         row_spacing=0,
         corner_radius=2
     ),
+    "7120-25": SheetLayout(
+        display_name="7120-25",
+        page_size=PAPER_SIZES["A4"],
+        label_width=35,
+        label_height=35,
+        columns=5,
+        rows=7,
+        column_spacing=5,
+        row_spacing=5,
+        corner_radius=0
```

Comment:
Thanks for the contribution, I will merge this soon.

According to some [images I found online](https://www.avery-zweckform.com/produkt/quadratische-etiketten-l7120-25) this label seems to have rounded corners, while others show it has sharp corners.
Can you confirm the corners are sharp? Could you maybe also send a picture of a physical label?

Reply:
At least my version has sharp corners:
![7120](https://github.com/user-attachments/assets/d0665924-6dab-41f6-9e8e-29b2fdc23a2b)

Reply:
Thanks for the info, I will merge this now. Have also fixed and tested InvenTree 0.16.x support, should now work with that version as well.